src/components/GameList/GameList.tsx

```diff
@@ -185,6 +188,7 @@ export class GameList extends React.PureComponent<GameListProps, any> {
                             white={game.white}
                             player={this.props.player}
                             gobanref={(goban) => game.goban = goban}
+                            size={game.height}
```

Comment:
We should probably do width x height as we do allow for non square games, also that's what we do everywhere else so in the name of consistency if nothing else :)

Reply:
:-o  I had no idea you could play non-square games. I did think it was weird that there was height and width, but I didn't think about it too much.

What about putting a single number for square games, and something like "17x12" otherwise?

Reply:
Well i think for consistency we should keep with the NxN format regardless since it's what we do everywhere else (also I think it's a good UI practice as your eyes can scan for that easily enough.. having two formats for the same information might be a bit jarring I'd think)